src/modules/account/currency-api.js

```diff
@@ -50,3 +51,28 @@ export class CurrencyTools extends Bridgeable<EdgeCurrencyTools> {
     return this.changeSettings(settings)
   }
 }
+
+export class ExchangeTools extends Bridgeable<EdgeExchangeTools> {
+  _settings: Object
+
+  constructor () {
+    super()
+    this._settings = {}
+  }
+
+  get exchangeInfo () {
```

Comment:
Is this going to barf since there is no return type.. I think this was part of what I got back today as a flow error.